src/main/resources/webinterface/css/simple-sidebar.css

```diff
@@ -113,4 +113,66 @@
     #wrapper.toggled #sidebar-wrapper-dwarfs {
         width: 250px;
     }
+}
+
+/* Sidebar Icons */
+#sidebar-wrapper-elves .carrotAction {
+	background-image: url("../icons/carrot.png");
+	background-repeat: no-repeat;
+	background-position: 10%;
```

Comment:
I thought about doing this, but decided against it as it would make the class structure more convoluted. I'll try it out and see how complicated it becomes.